vortex-tui/src/browse/mod.rs

```diff
@@ -41,51 +40,53 @@ enum HandleResult {
 fn handle_normal_mode(app: &mut AppState, event: Event) -> HandleResult {
     if let Event::Key(key) = event {
         if key.kind == KeyEventKind::Press {
-            match key.code {
-                KeyCode::Char('q') => {
+            match (key.code, key.modifiers) {
+                (KeyCode::Char('q'), _) => {
```

Comment:
what, you don't wanna implement `C-x C-c` ??

Reply:
I truly sincerely do not